ketcher-autotests/tests/Indigo-Tools/Layout/layout.spec.ts

```diff
@@ -56,6 +96,111 @@ test.describe('Indigo Tools - Layout', () => {
       page,
     );
     await selectTopPanelButton(TopPanelButton.Layout, page);
-    await takeEditorScreenshot(page);
+  });
+
+  test('Sprout bonds to the structure after Layout', async ({ page }) => {
+    /*
+    Test case: EPMLSOPKET-1806
+    Description: User is able to change the structure: sprout the bonds, change the atom symbols, 
+    change the atoms/bonds properties after the Layout action.
+    */
+    const x = 300;
+    const y = 300;
+    const anyAtom = 0;
+    await openFileAndAddToCanvas('Molfiles-V2000/toluene.mol', page);
+    await waitForSpinnerFinishedWork(page, async () => {
+      await selectTopPanelButton(TopPanelButton.Layout, page);
+    });
+    await moveOnAtom(page, 'C', anyAtom);
+    await dragMouseTo(x, y, page);
+    await waitForSpinnerFinishedWork(page, async () => {
+      await selectTopPanelButton(TopPanelButton.Layout, page);
+    });
+    await selectAtomInToolbar(AtomButton.Oxygen, page);
+    await clickOnAtom(page, 'C', anyAtom);
+  });
+
+  test('Layout action on part of structure with S-Group', async ({ page }) => {
+    /*
+    Test case: EPMLSOPKET-1812
+    Description: The Layout action is implemented for the whole canvas.
+    */
+    await openFileAndAddToCanvas('Molfiles-V2000/distorted-Sgroups.mol', page);
+    await selectPartOfMolecules(page);
+    await selectTopPanelButton(TopPanelButton.Layout, page);
+  });
+
+  test('Layout action on part of structure with R-Group label', async ({
```

Comment:
For this kind of test you can create a separate `test.describe` block and make a loop over the tests – they are just the same and only file name changes.